frontend/src/components/pages/CampCreation/CampDetails/index.tsx

```diff
@@ -1,49 +1,520 @@
+                value={campCapacity}
+                borderColor={!campCapacity && showErrors?"red":"gray.200"}
+                borderWidth={!campCapacity && showErrors?"2px":"1px"}onChange={handleCampCapacity}/> 
+              {errorText(campCapacity,"You must enter a number.")} 
+            </Box>
+        </Box>
+      </HStack>
+      
+      <Checkbox marginTop="8px" onChange={toggleEDLP}>
+        <Text textStyle="buttonSemiBold">
+          Camp offers early drop-off and late pick-up
+        </Text>
+      </Checkbox>
+      
+
+      {offersEDLP?(
+        <>
+          <HStack alignItems="start" spacing={4} marginTop="24px">
+            <Box width="160px">
+              <Text textStyle="buttonSemiBold"> 
+                Earliest Drop-off{" "}
+                <Text textStyle="buttonSemiBold" display="inline" color="red">*</Text>
+              </Text>
+              
+              <Input type="time" placeholder='XX:XX' width="160px" height="52px" marginTop="8px"
+                value={earliestDropOffTime}
+                borderColor={!earliestDropOffTime && showErrors?"red":"gray.200"}
+                borderWidth={!earliestDropOffTime && showErrors?"2px":"1px"}onChange={handleEarliestDropOffTime}/> 
+              {errorText(earliestDropOffTime,"You must specify a time.")}
+            </Box>
+            
+            <Text paddingTop="45px"> to </Text>
+            
+            <Box width="160px">
+              <Text textStyle="buttonSemiBold">
+                Latest Pick-up{" "}
+                <Text textStyle="buttonSemiBold" display="inline" color="red">*</Text>
+              </Text>
+              <Input type="time" placeholder='XX:XX' width="160px" height="52px" marginTop="8px"
+                value={latestPickUpTime}
+                borderColor={!latestPickUpTime && showErrors?"red":"gray.200"}
+                borderWidth={!latestPickUpTime && showErrors?"2px":"1px"}onChange={handleLatestPickUpTime}/> 
+              {errorText(latestPickUpTime,"You must specify a time.")}
+            </Box>
+          </HStack>
+          
+          <Text textStyle="buttonSemiBold" marginTop="24px">
+            EDLP Price Per 30 Minutes{" "}
+            <Text textStyle="buttonSemiBold" display="inline" color="red">*</Text>
+          </Text>
+        
+          <InputGroup width="275px" marginTop="8px">
+            <InputLeftElement pointerEvents='none'color='black'fontSize='1.2em'>
+              $
+            </InputLeftElement>
+              <Input type="number" placeholder='0.00'
+                value={priceEDLP}
+                borderColor={!priceEDLP && showErrors?"red":"gray.200"}
+                borderWidth={!priceEDLP && showErrors?"2px":"1px"}
+                onChange={handlePriceEDLP}/>
+          </InputGroup>
+          {errorText(priceEDLP,"You must add a fee.")}
+        </>
+      ):(<></>)
+      }
+      <Box width="50%" maxWidth="870px" bg="#8C9196" height="2px" margin="40px 0"/>
```

Comment:
would it be better to wrap the contents above in a box and then add margin to that? this seems a little fragile lol